wavefront/values.yaml

```diff
@@ -66,7 +66,7 @@ collector:
   ## Required if `collector.useProxy` is true and `proxy.enabled` is false
   # proxyAddress: wavefront-proxy:2878
 
-  ## Enable metrics for control plane dashboard (default false)
+  ## Enable metrics for control plane dashboard (default tue)
```

Comment:
@mmichael  
Also, I read this comment as default value is false unless explicitly specified as "enabled:true". So why change it ?

Also if we really wanted to go ahead with this to reduce confusion, then spell check "tue" instead of "true"

Reply:
shoot, it did / initial review did miss the typo